packages/front-end/components/Layout/TopNav.tsx

```diff
@@ -227,6 +227,17 @@ const TopNav: FC<{
                 </>
               )}
               <div className="dropdown-divider"></div>
+              <Link href={"/settings/personal-access-tokens"}>
```

Comment:
The `/settings/` path today is specifically for admin-only settings.  Maybe we should use a new top-level path like `/account/`?

Reply:
moved to `/account`